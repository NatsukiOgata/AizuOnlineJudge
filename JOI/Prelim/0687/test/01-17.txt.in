100
66 34 15 69 87 91 4 89 98 32 6 80 83 19 14 44 37 29 55 62 12 2 94 46 72 88 84 73 63 48 100 53 22 42 51 59 77 92 90 36 75 28 52 79 5 16 78 67 7 57 64 65 9 11 25 23 43 68 74 41 86 33 27 47 18 30 81 24 45 85 56 93 31 39 21 95 96 54 10 70 71 17 58 99 49 1 61 60 97 40 38 35 50 26 82 3 76 8 20 13
