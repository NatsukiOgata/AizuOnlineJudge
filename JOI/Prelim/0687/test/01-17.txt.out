1596
3354
